### PR TITLE
[IT-1178] Setup SSO access to AWS scipool accounts

### PR DESCRIPTION
**Pull request by zaro0508**
Setup to allow users in the JC scipool admin and scipool community manager
groups access to AWS scipooldev and scipoolprod accounts.  We setup the
admin and a community manager roles.

depends on https://github.com/Sage-Bionetworks/aws-infra/pull/303

### CODE REVIEW

File: org-formation/700-aws-sso/_tasks.yaml [hunk 1]
```diff
@@ -30,6 +30,126 @@ Parameters:
...
+    instanceArn: !Ref instanceArn
+    principalId: !Ref scipoolDevAdminGroup
+    permissionSetName: 'Administrator'
+    managedPolicies: [ 'arn:aws:iam::aws:policy/AdministratorAccess' ]
+    sessionDuration: 'PT8H'
+
+SsoScipoolProdAdmin:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.6/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-scipoolprod-community-manager'
```
Comment (brucehoff) [inline org-formation/700-aws-sso/_tasks.yaml:138]: shouldn't this be:
```
StackName: !Sub '${resourcePrefix}-${appName}-scipoolprod-admin'
```
?

**Review by brucehoff — CHANGES_REQUESTED** [review]

File: org-formation/700-aws-sso/_tasks.yaml (hunk 1 above)
Comment (zaro0508) [reply org-formation/700-aws-sso/_tasks.yaml]: yes, fixed.  thanks.

**Review by brucehoff — APPROVED** [review]

**Review by ahayden — CHANGES_REQUESTED** [review]
Shouldn't stridespool be part of this change?

**Comment by zaro0508** [comment]
no @ahayden , strides account is not under our organization account therefore we cannot manage it with org-formation.  We have the SAML integration for strides and we'll probably need to keep it like that.

**Comment by ahayden** [comment]
We will still have strides through Jumpcloud SAML directly (not AWS SSO).

**Review by ahayden — APPROVED** [review]
Looks good

**Pull request merged**